app/src/main/java/com/hemdan/mvvm/ui/main/actorslist/ActorsListFragment.kt

```diff
@@ -0,0 +1,62 @@
+package com.hemdan.mvvm.ui.main.actorslist
+
+import android.os.Bundle
+import android.view.View
+import androidx.lifecycle.Observer
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.hemdan.mvvm.R
+import com.hemdan.mvvm.data.model.PopularInfo
+import com.hemdan.mvvm.ui.base.BaseFragment
+import kotlinx.android.synthetic.main.fragment_actors_list.*
+import java.util.ArrayList
+import javax.inject.Inject
+
+class ActorsListFragment : BaseFragment() {
+
+    private lateinit var listAdapter: ActorsAdapter
```

Comment:
@MonaAliMahmoud if you followed the first comment on the adapter file above you will see that you can now create adapter like this var listAdapter = ActorsAdapter() by using this way you won't have to use the lateinit which may cause crash in some case the adapter is not initialized and used